schedulelist.h

```diff
@@ -0,0 +1,20 @@
+#ifndef SCHEDULELIST_H
+#define SCHEDULELIST_H
+
+#include <QString>
+
+class ScheduleList
```

Comment:
Seems to me that this class name doesn't suit to the defined set of fields. Maybe, StudentDay?

Reply:
It's the scope of another Pull Request